components/forms/w-field.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <component :is="as" class="field" :class="{ 'is-invalid': hasErrorMessage, 'is-disabled': disabled, [$attrs.class || '']: true }" :role="role" v-bind="wrapperAria">
-    <component :is="labelType" v-if="label" class="field-label" :id="labelId" :for="labelFor" :role="valueOrUndefined(labelLevel, 'heading')" :aria-level="valueOrUndefined(labelLevel, labelLevel)">{{ label }}<span v-if="optional" class="pl-8 font-normal text-14 text-gray-500"> (valgfritt)</span></component>
+  <component :is="as" class="relative" :class="{[input.invalid]: hasErrorMessage, [$attrs.class || '']: true}" :role="role" v-bind="wrapperAria">
```

Comment:
>`'is-disabled': disabled`

I see you removed this but I couldn't find where that was applied using component-classes. Did that class do anything?

Reply:
It is here now https://github.com/warp-ds/vue/pull/11/files/a9c13a0bacc8734ab4bd6342a161f5fa3ed3da14#diff-76fbf39766e49e08a1ba36c837d5476581c1aad78c039651151f402a34386f29R51 on `input`